squad/frontend/templates/squad/tests.jinja2

```diff
@@ -29,11 +29,11 @@
         </thead>
         {% if results %}
             {% for test in results %}
-            {% set test_details_url = url('testrun_suite_test_details', args=[build.project.group.slug, build.project.slug, build.version, test.test_run.id, test.suite.slug.replace('/', '$'), test.name]) %}
+            {% set test_history_url = url('test_history', args=[build.project.group.slug, build.project.slug, test.name]) %}
```

Comment:
this doesn't work. I'm getting 404 when I click the link. Beside that, this link should lead to test details, not history. So you went one step too far :)

Reply:
@mwasilew btw, this used to point to history before we added the details page, some rework needs to happen to the TestResults class then. lemme check the 404, you got that on the android project?